firefox_ui_tests/remote/security/test_mixed_content.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestMixedContent(FirefoxTestCase):
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_url = 'https://mozqa.com/data/firefox/security/mixedcontent.html'
+
+    def test_mixed_content(self):
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.test_url)
+
+        def check_favicon_image(mn):
+            favicon_image = mn.execute_script("\
+                return arguments[0].ownerDocument.defaultView\
+                                   .getComputedStyle(arguments[0])\
+                                   .getPropertyValue('list-style-image');\
+            ", script_args=[self.browser.navbar.locationbar.favicon])
+            return 'identity-icons-https-mixed-display' in favicon_image
+
+        self.wait_for_condition(check_favicon_image)
```

Comment:
Please replace the `wait_for_condition()` call with `Wait(...).until(...)`. For existing tests we are undergoing a maintenance soon via [bug 1145112](https://bugzilla.mozilla.org/show_bug.cgi?id=1145112)